frontend/src/components/Shared/footer/Content.jsx

```diff
@@ -96,9 +95,14 @@ const Nav = () => {
     },
     {
       name: 'Tiktok',
-      link: 'https://www.tiktok.com/@sipnplaynycofficial?lang=en',
+      link: 'https://www.tiktok.com/@sipnplaynyc?lang=en',
       icon: <FaTiktok />,
     },
+    {
+      name: 'GitHub',
+      link: 'https://github.com/devxMani/PlayCafe',
```

Comment:
add this link here:-
https://github.com/RamakrushnaBiswal/PlayCafe

Reply:
@RamakrushnaBiswal done added